### ISSUE TITLE
BehaviorSpace 'collect results every n ticks' does not work on version 6.1.0

### ISSUE THREAD

**Issue by pietropollo** [NetLogo/NetLogo#1775]
I need to collect results of my model only every n ticks. The NetLogo FAQ (http://ccl.northwestern.edu/netlogo/faq.html) states a solution: on the go commands, simply change "go" to "repeat n [ go ]" (n being a number). However, this does not work in the current version 6.1.0, regardless of the model (using any model from the library) or computer in which is run. The output using "repeat n [ go ]" is the same as using simply "go". This is true if the "measure runs at every step" box is either checked (collects results of all steps) or unchecked (collects results only of the final step of the simulation). According to a colleague, this used to worked on an older version of NetLogo.

**Comment by LaCuneta** [comment]
Hey Pietro,

Thanks for reporting this issue, and sorry you ran into it!

I took the Wolf Sheep Predation model from the library in NetLogo 6.1.0, made a new BehaviorSpace experiment, changed the `Go commands:` to `repeat 10 [ go ]`, then changed the `Measure runs using these reporters:` to `ticks` and `count turtles` (on separate lines).  I added a `Time limit` of 200 and saved and ran the experiment.  When I open the outputted spreadsheet file, I see the column for `ticks` increasing by 10 at a time along with the changing `count turtles`.  

So it looks to me like the process from the FAQ is still working in 6.1.0 at least in some cases.  Do those sound like the steps you followed?  If not, can you described what you did differently to setup your model?  Or email your model in to bugs@ccl.northwestern.edu (we won't share it outside the development team).  If so, I'll take a look and see what's going wrong.  

Thanks, again.

**Comment by pietropollo** [comment]
Hi there,

Thanks for replying! I thought that the output with the "measure runs at every step" box checked was still the same regardless of using "repeat". However, I took that conclusion because I thought "step" was a synonymous for "tick". In your example (using "ticks" as a reporter) I realized that each step is actually each interval. Thank you for clarifying and so sorry for the misunderstanding!

Cheers
